layouts/partials/tw/impact-carousel-panels.html

```diff
@@ -0,0 +1,117 @@
+<!-- Panels -->
+<div
+  class="relative col-span-8 col-start-1 row-span-3 row-start-1 w-full items-center justify-center md:col-span-14"
+>
+  <!-- PANEL NAVIGATION -->
+  <button
+    class="absolute left-0 top-0 z-20 h-full w-16 text-6xl text-white"
+    @click="select($el.id) - 1"
```

Comment:
Should that be `select($el.id - 1)`? I think select() doesn't return anything, so it would just be the same as `undefined - 1`. 

 I'm not sure `select($el.id - 1)` will work either because id is a string and not a number. Worth a shot though. 

In the featured/slider, I have currentSlide be a number and then there is `this.$watch("currentSlide"` call that tells Alpine to scroll over to whatever currentSlide gets set to whenever it changes.

That one cheats a bit by making the arrow buttons not actually do anything. Instead when you click an arrow, it falls through to the underlying slide and then that slide gets the click and brings itself into view. But that aside, I think having the currentPanel be a number makes some of the other stuff easier.